### PR TITLE
Update to latest cortex to fix duplicate labels

### LINKED ISSUE #1220: Long label name queries returns duplicate value
image

There must be a bug somewhere in the new schema.

### PR DESCRIPTION
**Pull request by cyriltovena**
Bring latest cortex with fetching label without duplicates across multiple store.

Fixes #1220

### CODE REVIEW

**Comment by cyriltovena** [comment]
Investigating the flush test.

**Comment by cyriltovena** [comment]
Alright lifecycler needs to be started now since this change https://github.com/cortexproject/cortex/commit/00566f61af49f2dd8257413d9e49a3053f708fce#diff-dcd65a4ff2de424be70a561f141e067bR180

**Comment by cyriltovena** [comment]
https://github.com/cortexproject/cortex/blob/master/pkg/ring/kv/memberlist/memberlist_client.go#L619
I think we lost ARM support because of this change.

File: go.mod [hunk 1]
```diff
@@ -73,3 +71,6 @@ replace k8s.io/client-go => k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
 
 // Override reference causing proxy error.  Otherwise it attempts to download https://proxy.golang.org/golang.org/x/net/@v/v0.0.0-20190813000000-74dc4d7220e7.info
 replace golang.org/x/net => golang.org/x/net v0.0.0-20190923162816-aa69164e4478
+
+// Override cortex to get ARM support.
+replace github.com/cortexproject/cortex => github.com/cyriltovena/cortex v0.0.0-20191114231131-3e991d87dfa9
```
Comment (pstibrany) [inline go.mod]: This is now fixed on Cortex master.

Comment (cyriltovena) [reply go.mod]: I know I was busy on something else Owen took over

**Pull request closed without merging**